index.html

```diff
@@ -1122,37 +1122,91 @@
-[=ancillary data=] at all.
+  <dt><dfn>Summarizing ancillary APIs</dfn></dt>
+  <dd>
+    APIs that filter, summarize, or time-shift information available from
+    [=non-ancillary APIs=], like the [[[event-timing]]] and <a
+    data-cite="intersection-observer#introduction">IntersectionObserver</a>.
+  </dd>
 
-Users may be willing to share [=ancillary data=] if it is aggregated with
-the data of other users, or [=de-identified=]. This can be useful
-when [=ancillary data=] contributes to a collective benefit in a way
-that reduces privacy threats to individuals (see <a href="#principle-collective-privacy">collective
-privacy</a>).
+  <dt><dfn>Novel ancillary APIs</dfn></dt>
+  <dd>
+    APIs that provide new information that's primarily useful to support the
+    ancillary uses, like <a data-cite="element-timing#sec-intro">element paint
+    timing</a>, <a data-cite="performance-measure-memory#intro">memory usage
+    measurements</a>, and <a
+    data-cite="deprecation-reporting#deprecation-report">deprecation
+    reports</a>.
+  </dd>
+</dl>
 
-<aside class="example">
-  Privacy-preserving measurement techniques may be used for aggregate calculations while minimizing
-  the number of actors that have access to personal data about many individual people. Encryption and
-  privacy-preserving proxies may minimize the number of actors that have access to personal data or
-  hide the contents of personal data. But even
-  with those protections, some people may prefer not to participate in some kinds of measurement.
+All of these sources of data can reveal [=personal data=] about a person's
+configuration, device, environment, or behavior that could be <a
+href="#hl-sensitive-information">sensitive</a> or be used as part of <a>browser
+fingerprinting</a> to <a data-lt="cross-context recognition">recognize people
+across contexts</a>. In order to uphold the principle of [[[#data-minimization]]], [=sites=] and
+[=user agents=] should seek to understand and respect people's goals and preferences about
+use of this data.
+
+[=User agents=] usually can't do much about data collection from the
+[=non-ancillary APIs=].
+
+There is disagreement about how [=user agents=] should handle [=summarizing
+ancillary APIs=]. Advocates of these APIs argue that they're hard to use to
+extract [=personal data=], they're more efficient than collecting the same
+information though [=non-ancillary APIs=], sites are less likely to adopt these
+APIs if a significant number of people turn them off, and that the act of
+turning them off can ironically contribute to [=browser fingerprinting=].
+Opponents argue that if data's easier or cheaper to collect, more sites will
+collect it, and because there's still some risk and cost, users should be able
+to turn off this group of APIs that probably won't directly break a site's
+functionality.
+
+We do have consensus that [[[#information]]] governs [=summarizing ancillary
+APIs=] and that:
```

Comment:
Do we need to note consensus that the other principles in the document still apply? That would seem to be the case for all principles.

Reply:
I'd be fine with dropping that.

Reply:
Reworded to

> Because different users are likely to have different preferences:

That's still a little awkward, so I'd welcome other wording suggestions.